libraries/fula-client/src/index.ts

```diff
@@ -1,165 +1,180 @@
 import type {SchemaProtocol} from "@functionland/file-protocol";
 import {FileProtocol} from '@functionland/file-protocol';
-import {PROTOCOL as GRAPH_PROTOCOL, Request, Result, submitQuery, submitSubscriptionQuery} from '@functionland/graph-protocol'
+import {
+  PROTOCOL as GRAPH_PROTOCOL,
+  Request,
+  Result,
+  submitQuery,
+  submitSubscriptionQuery
+} from '@functionland/graph-protocol'
 import {configure} from './config';
 import Libp2p, {constructorOptions, Libp2pOptions} from 'libp2p';
 import {Connection, Status} from "./connection"
+import debug from "debug";
+import PeerId from "peer-id";
+
+debug.disable()
```

Comment:
why disable debug here?

Reply:
I have to do it some where because it only manage able throw code or local storage variable and it consume a lot of resource in browser.
but you are right it think we should make an issue to add:
[env-build-blugin](https://www.npmjs.com/package/esbuild-envfile-plugin)
[.env](https://www.npmjs.com/package/dotenv)
and move logger somewhere more meaningfull